test/unit/services/test-get-export-csv.js

```diff
@@ -78,6 +78,10 @@ describe('services/get-export-csv', function () {
       expect(getExportCsv(orgUnit.TEAM.name, helper.SUSPENDED_LIFER_EXPORT_RESULT.suspendedLiferExportDetails, tabs.EXPORT.SUSPENDED_LIFERS_EXPORT).csv).to.eql(helper.SUSPENDED_LIFER_CSV.csv)
     })
 
+    it('for ARMS Export', function () {
```

Comment:
Can ARMS be changed to T2A please? Thanks

Reply:
That should be everything resolved now